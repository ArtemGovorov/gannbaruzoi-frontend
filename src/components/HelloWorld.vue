<template>
  <div></div>
</template>

<script>
export default {
  name: 'hello-world',

  props: {

  },
  data () {

  },
  watch: {

  },
  methods: {

  }
}
</script>

<style scoped>
</style>
