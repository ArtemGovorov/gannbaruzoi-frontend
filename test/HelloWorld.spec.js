import {shallow} from 'vue-test-utils'
import HelloWorld from '@/components/HelloWorld'

describe('TaskListItem.vue', () => {

  it('renders correct contents', () => {
    const wrapper = shallow(HelloWorld, {})
    expect(wrapper.html()).toMatchSnapshot()
  })
})
