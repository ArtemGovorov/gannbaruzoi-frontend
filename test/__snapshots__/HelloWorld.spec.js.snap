// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskListItem.vue renders correct contents 1`] = `<div></div>`;
